protos/encodings_v2_1.proto

```diff
@@ -112,6 +112,9 @@ message MiniBlockLayout {
   // The page already records how many rows are in the page.  For mini-block we also need to know how
   // many "items" are in the page.  A row and an item are the same thing unless the page has lists.
   uint64 num_items = 9;
+
+  // Since Lance 2.2, we support miniblocks with larger chunk size (>= 64KB)
+  bool support_large_chunk = 10;
```

Comment:
Nit: Instead of `support_large_chunk` can we call this `has_large_chunk` or `has_large_chunks` (here and pretty much everywhere else)?